frontend/src/views/admin/assignments/manipulate-assignment-confirmation.tsx

```diff
@@ -39,33 +40,24 @@ const withdrawModalColumn = [
     },
 ];
 
-function compareString(str1: string, str2: string) {
-    if (str1 > str2) {
-        return 1;
-    } else if (str1 < str2) {
-        return -1;
-    }
-    return 0;
-}
-
-function compareAssignment(a1: Assignment, a2: Assignment) {
-    return (
-        compareString(a1.position.position_code, a2.position.position_code) ||
-        compareString(
-            a1.applicant.last_name || "",
-            a2.applicant.last_name || ""
-        ) ||
-        compareString(a1.applicant.first_name, a2.applicant.first_name)
-    );
-}
-
-export function MultiWithdrawOfferConfirmation(props: {
+export function MultiManipulateOfferConfirmation(props: {
     data: Assignment[];
     visible: boolean;
     setVisible: (visibility: boolean) => void;
-    withdrawOffers: () => void;
+    manipulateOffers: () => void;
```

Comment:
Does this typecheck? Also what does `manipulateOffers` do? Is it actually a callback? If so, call it `callback`